vendor-bin/owncloud-codestyle/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "config" : {
+    "platform": {
+      "php": "5.6.37"
```

Comment:
I wonder why this has been done and copy-pasted like this to lots of app repos?

For "real" dependencies that are used by the actual app (or core stable10) we have been mentioning PHP 5.6 so that whenever someone does ``composer update``, even though they may be currently running PHP  7.*, composer will try to find a combination of dependencies that will support PHP 5.6

In this case we do not really care - ``composer`` can find the necessary dependencies to go with whatever PHP  version we happen to be running at the time we do code style check.

Reply:
Please point me towards the initial starting point - the platform limitation here shouldn't have happened. 

Maybe someone had issues locally when developing on 7.x and then switching to 5.6 ?

Reply:
That will be difficult to pinpoint. But I notice that ``data_exporter`` has the very simple:
```
{
    "require": {
        "owncloud/coding-standard": "^1.0"
    }
}
```
So I guess pinning the PHP version snuck in somewhere.

Reply:
I removed the ``config`` ``platform`` section. CI still passes.